package.json

```diff
@@ -53,7 +53,38 @@
           "description": "Run the linter on save (onSave) or on type (onType)."
         }
       }
-    }
+    },
+    "languages": [
```

Comment:
This is still necessary.  For files without a `.json` extension, VSCode defaults to "Plain Text" language mode, which means it does not use the JSON schema.